lib/justice_gov_sk/infrastructure.rb

```diff
@@ -0,0 +1,19 @@
+module JusticeGovSk
+  module Infrastructure
+    # TODO implement async?
+
+    def crawl
+      (1..pages).each do |page|
+        url = uri.build(page: page)
```

Comment:
rozmyslal som prave nad tym, ze url vs uri, a akurat mi to vyslo takto, cize trochu aj obsadene a trochu aj premyslene